### PR TITLE
Add support for the rak10701 board, no touch

### PR DESCRIPTION
**Pull request by HookdomPonix**
Add support for the RAK10701 Field Tester device.  You must flash from the RUI3 firmware to Arduino before use.  

The TFT and the button on the side work, but the touch screen will require more work as its not the same libs as the ESP32 TFT/touch devices.

### CODE REVIEW

File: src/graphics/TFTDisplay.cpp [hunk 1]
```diff
@@ -484,16 +496,21 @@ bool TFTDisplay::connect()
...
 #elif defined(T_DECK) || defined(PICOMPUTER_S3)
     tft.setRotation(1); // T-Deck has the TFT in landscape
 #elif defined(T_WATCH_S3)
     tft.setRotation(2); // T-Watch S3 left-handed orientation
 #else
     tft.setRotation(3); // Orient horizontal and wide underneath the silkscreen name label
-#endif
     tft.fillScreen(TFT_BLACK);
+#endif
     return true;
```
Comment (caveman99) [inline src/graphics/TFTDisplay.cpp:514]: That change eliminates the fillScreen call for all other variants which is a break. What happens if you put the call back after the endif and remove it in the RAK14014 condition?

**Review by caveman99 — CHANGES_REQUESTED** [review]
See comments inline please.

File: variants/rak10701/platformio.ini [hunk 2]
```diff
@@ -0,0 +1,18 @@
+; The very slick RAK wireless RAK10701 Field Tester device.  Note you will have to flash to Arduino bootloader to use this firmware.  Be aware touch is not currently working.
+[env:rak10701]
+extends = nrf52840_base
+board = wiscore_rak4631
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak10701 -D RAK_4631
+  -L "${platformio.libdeps_dir}/${this.__env__}/BSEC2 Software Library/src/cortex-m4/fpv4-sp-d16-hard"
+  -DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/rak4631> +<mesh/eth/> +<mesh/api/> +<mqtt/>
+lib_deps =
```
[the remark this answers is not in the document]
Comment (HookdomPonix) [reply variants/rak10701/platformio.ini:9]: Fixed in the latest commit.  Thanks for the heads up.

File: src/graphics/TFTDisplay.cpp (hunk 1 above)
Comment (HookdomPonix) [reply src/graphics/TFTDisplay.cpp:515]: Thanks for the catch.   Fixed and recompiled/works on the 10701.

**Review by caveman99 — APPROVED** [review]

**Review by thebentern — APPROVED** [review]

**Comment by geeksville** [comment]
This pull request has been mentioned on **Meshtastic**. There might be relevant details there:

https://meshtastic.discourse.group/t/rak10701-field-tester-as-a-meshtastic-device/8406/7

**Pull request merged**